### DISCUSSION TITLE
How to append onto a given logger version when resuming from checkpoint

### DISCUSSION THREAD

**Discussion by escorciav** [Lightning-AI/pytorch-lightning#6799] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi,

I noticed that PL logger** creates a `version_X` folder inside the `lightning_logs` folder when I dispatch the Trainer*.

1. Is it possible to append new files, such as tfevent, config yaml, etc., onto a given `version_X` folder?
  Particularly, when using the `resume_from_checkpoint` argument of the Trainer?
  
2. (*) I noticed that the `lightning_logs` folder is also created during my inference. What's the responsible class 😊 ?

Thanks in advance!
Victor

** I guess. I'm also using Hydra, but I don't think the code delegates that to that package.

**Comment by awaelchli** [comment]
Hi, we are working on improving the state saving for loggers so that they can resume from the correct version.
https://github.com/PyTorchLightning/pytorch-lightning/issues/5342

For 2) yes, this gets created because also during inference you can log so the logger just creates this dir. Not sure how to avoid it without removing the logger completely
